cudaaligner/src/aligner_global_myers_banded.cpp

```diff
@@ -354,6 +375,19 @@ void AlignerGlobalMyersBanded::reset()
     alignments_.clear();
 }
 
+void AlignerGlobalMyersBanded::reset_bandwidth(const int32_t max_bandwidth)
+{
+    assert(max_device_memory_ >= 0);
```

Comment:
What happens if `max_device_memory_ == 0` or small? Does the algorithm take care of that?

Reply:
max_device_memory is set by the constructor, which ensures that it `max_device_memory_ >= 0`.
If `max_device_memory_ == 0`, `add_alignment` will always fail and return the error `StatusType::exceeded_max_alignments`.

Reply:
Good, so that case is taken care of.